modules/admin/proc-postgresql-certificate-for-crunchy-database.adoc

```diff
@@ -0,0 +1,38 @@
+[id='postgresql-certificate-for-crunchy-database_{context}']
```

Comment:
@shipsing - why are we documenting guidance specific to CrunchyDB?  The Postgres guidance should be vendor agnostic. We are not preferencing any one vendor over any other, there is no certification program.  For obtaining a TLS certificate, customers should refer to their Postgres vendor documentation.

Reply:
The content is added based on the inputs provided in this [JIRA](https://issues.redhat.com/browse/RHIDP-1238), which covers how to set up a TLS connection with Postgres. 
I agree customers should refer to their Postgres vendor documentation, but even for AWS RDS and Azure database, we have covered basic steps for our users to help them generate the required certificate files. Since having those certificate files a kind of prerequisites to proceed with the actual procedure, we have covered the basic details in these sections: Downloading PostgreSQL certificate for AWS RDS, Generating PostgreSQL certificate for Azure database, and Generating PostgreSQL certificate for the Crunchy database.

Reply:
Please refer to it's [Feature Link Jira](https://issues.redhat.com/browse/RHIDP-453)

> test multiple impelmentation of a PSQL version (Crunchy, RDS, Azure version)
we don't need to document how to setup Azure, RDS etc... that is customer responsibilty
we need to document where to provide connection information (username, password, host, port, tls certs etc...)

Customer is responsible for obtaining TLS certificates.  Once we start documenting it for vendors the expectation is that we're providing a higher level of support for some vendors over others.  Customers then request guidance for a vendor where we have no documentation and we've taken on the burden of maintaining all the guidance each and every time there is a change in vendor process.  We should not be documenting vendor specifics.

Reply:
Based on the coversation, going to remove these three sections from the doc: 

- Downloading PostgreSQL certificate for AWS RDS
- Generating PostgreSQL certificate for Azure database
- Generating PostgreSQL certificate for the Crunchy database